application/Profile/DBProfile.py

```
@@ -67,13 +71,21 @@
-        self.resend_cooldown = TTLCache(maxsize=1024, ttl=RESEND_COOLDOWN_TTL_SECOND)
+        self.resend_cooldown = TTLCache(maxsize=size, ttl=RESEND_COOLDOWN_TTL_SECOND)
+        logger.debug("resend_cooldown cache set up with %d, expiration time = %d", size, RESEND_COOLDOWN_TTL_SECOND)
 
-        with open('/var/www/ictrl/application/resources/activation_email_template.html', 'r') as f:
-            self.activation_email_body_template = f.read()
+        activation_email_template = '/var/www/ictrl/application/resources/activation_email_template.html'
+        logger.debug("Opening %s in read-only mode", activation_email_template)
+        try:
+            with open(activation_email_template) as f:
+                self.activation_email_body_template = f.read()
+        except IOError as e:
```

Ruff (0.8.2)

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)